solvexia_sdk/user/user.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from solvexia_sdk import api
+
+class user:
+    def __init__(self, user_id):
+        self.user_id = user_id
+    
+    def get_user_list(self):
+        response = requests.get(api.base_url + "users", headers=api.access_token)
```

Comment:
Just wondering if there is a specific reason why get_user_list() uses requests.get() instead of api.api_get like the other methods in this class.

Reply:
Updated it to be consistent. @nullfrit